### DISCUSSION TITLE
3차 중간점검 내용 wiki, notion 업데이트 완료🚀

### DISCUSSION THREAD

**Discussion by kimInDa** [FRONTENDSCHOOL6/react-project-11#108] [Announcements]
늦어서 죄송합니다.🙇
3차 중간점검 내용을 wiki와 notion에 업데이트 하였습니다.

디자인을 바꿔봤는데 마음에 들어서 앞서 작성한 회의록도 모두 바꿀 예정입니다.
예전엔 wiki에 쓴 걸 notion에 붙여넣기 하면 이모티콘이 전부 이상한 문자로 바뀌어 들어가고 레이아웃도 흐트러져서 2차 편집의 고난을 겪곤 했는데 지금은 고대로 들어가서 너무 좋네요😆
**언제나처럼 추가, 편집 대환영 입니다👍**

<br>

![screencapture-github-FRONTENDSCHOOL6-react-project-11-wiki-Sptint-Meeting-2023-9-19-3-2023-09-23-01_29_34](https://github.com/FRONTENDSCHOOL6/react-project-11/assets/105577805/bb948c0f-d9d4-453d-87f6-abcb017d0d44)

<br>

**Comment by summer787** [comment]
늘 꼼꼼히 정리하고 작성해주셔서 감사합니다!
